template/client/src/main/java/com/microsoft/azure/template/Hello.java

```diff
@@ -5,8 +5,15 @@
  */
 package com.microsoft.azure.template;
 
+/**
+ * This is a sample class for template app.
+ */
 public class Hello {
-    public String getMessage() {
+    /**
+     * Sample method.
+     * @return message
+     */
+    public final String getMessage() {
         return "hello";
     }
 }
```

Comment:
This, and the package-info.java below, are unrelated changes, aren't they?

Reply:
They both caused checkstyle and because of that the build fail.